<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocReaderQuizzesClass</class>
 <widget class="QWidget" name="DocReaderQuizzesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>222</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Quizzes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="authenticationPage">
      <layout class="QGridLayout" name="loginGridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="loginBtn">
         <property name="text">
          <string>Login with Google</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="developersLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>78</red>
               <green>110</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>78</red>
               <green>110</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Program from DocReader team</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="logoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="applicationPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="formTitleLabel">
           <property name="toolTip">
            <string>The title of form.</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Form Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="documentTitleLabel">
           <property name="toolTip">
            <string>The name of the form in Google Drive. Each form is a file in Google Drive and this file has a name. This field sets this name.</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Document Title</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="formDescriptionLabel">
           <property name="toolTip">
            <string>The description of the form.</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>Form Description</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="formTitleLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="formDocumentLineEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="formDescriptionLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="chooseFileBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Choose a file to be converted to a Google Form.</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Choose File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="processPdfBtn">
         <property name="toolTip">
          <string>Process the PDF using AI. The outcome is a textual representation of the quiz. This representation is used by the app to create a Google Form.</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Process PDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createQuizBtn">
         <property name="toolTip">
          <string>Create a quiz from the chosen file.</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Create Quiz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>75</red>
               <green>117</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>75</red>
               <green>117</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled/>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Segoe Print</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Program from DocReader team</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
